### PR TITLE
fix(condo): DOMA-12475 fix commission calc

### PR DESCRIPTION
**Pull request by vovaaxeapolla**
## Summary by CodeRabbit

* **Bug Fixes**
  * Fee distribution now respects billing category when calculating and allocating fees, improving accuracy for category-scoped receipts.

* **Tests**
  * Added tests covering category-specific fee distribution scenarios to validate implicit and explicit fee/service-charge calculations and ensure existing amount computations remain stable.

### CODE REVIEW

File: apps/condo/domains/acquiring/schema/CalculateFeeForReceiptService.test.js [hunk 1]
```diff
@@ -46,7 +47,24 @@ describe('CalculateFeeForReceiptService', () => {
             expect(result.explicitFee).toBe('0')
             expect(result.explicitServiceCharge).toBe('0')
         })
+
+        test('Check the calculations for implicit fees with category', async () => {
```
Comment (pahaz) [inline apps/condo/domains/acquiring/schema/CalculateFeeForReceiptService.test.js]: Could you please test more edge cases for this logic?
